### PR TITLE
default security groups should not be attached

### PR DESCRIPTION
**Pull request by pravinva**
AWS Proserve Mission 500, config rules:

Default security groups should not be attached to EC2,RDS,ELB

### CODE REVIEW

**Comment by msambol** [comment]
Hi @pravinva. One of my customers is interested in this too, but also when the group name starts with `launch-wizard`. Is that something I can add? Or should I create another rule?

File: python/default-security-group-attached [hunk 1]
```diff
@@ -0,0 +1,81 @@
+# This file made available under CC0 1.0 Universal (https://creativecommons.org/publicdomain/zero/1.0/legalcode)
+# 
+# Description: Check that default security groups are not attached to EC2,RDS,ELB.
+#
+# Trigger Type: Change Triggered
+# Scope of Changes: EC2:NetworkInterfaces
```
Comment (msambol) [inline python/default-security-group-attached]: This should be singular --> `EC2:NetworkInterfaces`

File: python/default-security-group-attached [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+#            "Resource": "*"
+#        }
+#    ]
+#}
+
+
+import boto3
+import json
+
+APPLICABLE_RESOURCES = ["AWS::EC2::NetworkInterfaces"]
```
Comment (msambol) [inline python/default-security-group-attached]: More importantly, here.

**Comment by pravinva** [comment]
@gregjinkim 
I wanted to check security groups attached to ELB as well, but now i realize its not in the 'applicable resources'. I'll use the groupname from security groups and resubmit. thanks

File: python/default-security-group-attached (hunk 1 above)
Comment (pravinva) [reply python/default-security-group-attached]: @msambol  thanks. will fix it

**Comment by pravinva** [comment]
Closing pull request. Comments have been incorporated in https://github.com/awslabs/aws-config-rules/pull/73

**Pull request closed without merging**